apps/fulfillment-e2e/readme.md

```diff
@@ -0,0 +1,11 @@
+## First run
+
+To run E2E tests locally you have to create `.env.local` file first.
+Use `.env.template` as an example.
+
+## Storfront E2E tests commands
```

Comment:
Storefront links and commands here

Reply:
Please, check `npm` commands in `package.json` file - I guess we will have to create same commands for FA